src/main/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -78,6 +80,13 @@ class StockMovementItem {
         return stocklistItem?.quantity ?: null
     }
 
+    BigDecimal getPacksRequested () {
+        if (packSize == null || packSize == 0) {
```

Comment:
so when we are checking if packSize is null or 0, we can use !packSize

Reply:
I made other changes but I'd prefer to leave this as it is now, I feel like this way is more explicit to show what are we checking instead of `!packSize`